package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ux-pattern-library",
-  "version": "0.2.0",
+  "version": "0.1.46",
```

Comment:
I think remove it for this PR. We should think about how and when to version, but let's not start here or have this slow this getting in. I'll ping ENG for any standards they have when versioning edX software. Thanks!